app/views/devise/registrations/_form.html.haml

```diff
@@ -4,18 +4,24 @@
   .form-group
     = f.label :email, 'Email address', class: 'control-label'
     %br/
-    = f.email_field :email, autofocus: true, class: 'form-control'
+    = f.email_field :email, autofocus: true, autocomplete: "email", class: 'form-control'
   .form-group
     = f.label :password, class: 'control-label'
     - if @minimum_password_length
       %em
         (#{@minimum_password_length} characters minimum)
     %br/
-    = f.password_field :password, class: 'form-control'
+    = f.password_field :password, autocomplete: "new-password", class: 'form-control'
   .form-group
     = f.label :password_confirmation, 'Re-Type Password', class: 'control-label'
     %br/
-    = f.password_field :password_confirmation, class: 'form-control'
+    = f.password_field :password_confirmation, autocomplete: "new-password", class: 'form-control'
   .form-group
     = f.submit "Sign up for Venue", class: 'form-submit'
-    = button_tag 'Cancel', class: 'form-submit', onclick: 'cancelModal(event);' 
+    = button_tag 'Cancel', class: 'form-submit', onclick: 'cancelModal(event);'
+  .social-login-buttons  
+    .facebook
```

Comment:
Is there a css class for `facebook`? I would like to see some screenshots in PR description

Reply:
There is. Added a view in the description.